caracara/modules/hosts/hosts_filters.py

```diff
@@ -268,6 +268,34 @@ def example(self) -> str:
             "OUs. Programmatically, you can pass a Python list of OUs."
         )
 
+class HostMACAddressFilterAttribute(FalconFilterAttribute):
+    """Filter by host's MAC address."""
+
+    name = "MACAddress"
+    fql = "mac_address"
+
+    def example(self) -> str:
+        """Show filter example."""
+        return (
+            "This filter accepts a MAC address string associated with a "
+            "network interface, e.g. 01-22-33-44-55-66. You can also"
+            "comma delimit strings for multiple matches."
+        )
+
+class HostDeviceIdFilterAttribute(FalconFilterAttribute):
+    """Filter by host's sensor id."""
+
+    name = "DeviceId"
+    fql = "device_id"
+
+    def example(self) -> str:
+        """Show filter example."""
+        return (
+            "This filter accepts a MAC address string associated with a "
```

Comment:
The help text here isn't quite right. I'll get this rewritten for `caracara-filter`.

Reply:
Thanks, updated.